Hunting Queries/GitLab/GitLab_MaliciousIP

```diff
@@ -0,0 +1,48 @@
+id: 7241740a-5280-4b74-820a-862312d721a8
+name: GitLab - TI - Connection from Malicious IP 
+description: |
+  'This query correlates Threat Intelligence data from Sentinel with GitLab NGINX Access Logs (available in GitLab CE as well) to identify access from potentially TI-flagged IPs.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ThreatIntelligence
+    dataTypes:
+      - ThreatIntelligenceIndicator
+  - connectorId: ThreatIntelligenceTaxii
+    dataTypes:
+      - ThreatIntelligenceIndicator
+   - connectorId: Syslog
+    dataTypes:
+      - Syslog
+queryFrequency: 1h
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Initial Access
+relevantTechniques:
+  - T1078
+query: |
+
+ThreatIntelligenceIndicator
```

Comment:
We probably need to check for the latest indicator as well as see if it is still active so that we do not provide wrong information by adding these to the query. 
  | summarize LatestIndicatorTime = arg_max(TimeGenerated, *) by IndicatorId
  | where Active == true

Please look and model the queries similar to this one: https://github.com/Azure/Azure-Sentinel/blob/master/Detections/ThreatIntelligenceIndicator/IPEntity_CustomSecurityLog.yaml

Reply:
I already take latest indicators in the query, something I missed?